packages/template-retail-react-app/app/components/product-scroller/index.jsx

```diff
@@ -80,43 +83,52 @@ const ProductScroller = forwardRef(
                             ...scrollProps?.sx
                         }}
                     >
-                        {(isLoading ? [0, 1, 2, 4] : products).map((product, idx) => {
-                            return (
-                                <Box
-                                    key={product?.id || idx}
-                                    flex="0 0 auto"
-                                    width={itemWidth}
-                                    style={{scrollSnapAlign: 'start'}}
-                                >
-                                    {isLoading ? (
-                                        <Stack data-testid="product-scroller-item-skeleton">
-                                            <AspectRatio ratio={1}>
-                                                <Skeleton />
-                                            </AspectRatio>
-                                            <Stack spacing={2}>
-                                                <Skeleton width="150px" height={5} />
-                                                <Skeleton width="75px" height={5} />
-                                            </Stack>
-                                        </Stack>
-                                    ) : (
-                                        <ProductTile
-                                            data-testid="product-scroller-item"
-                                            product={product}
-                                            {...(typeof productTileProps === 'function'
-                                                ? {...productTileProps(product)}
-                                                : {...productTileProps})}
-                                            dynamicImageProps={{
-                                                widths: ['70vw', '70vw', '40vw', '30vw']
-                                            }}
-                                        />
-                                    )}
-                                </Box>
-                            )
-                        })}
+                        {isLoading
+                            ? [0, 1, 2, 4].map((key) => {
```

Comment:
```suggestion
                            ? [0, 1, 2, 3].map((key) => {
```

Reply:
The original array was 0 1 2 4, but the values aren't used, so it doesn't really matter.